tests/testcases/sort/test_01/test.vcf

```diff
@@ -0,0 +1,245 @@
+##INFO=<ID=known_TSA,Number=1,Type=String,Description="Type of sequence alteration. Child of term sequence_alteration as defined by the sequence ontology project.">
+##INFO=<ID=known_CLIN_uncertain_significance,Number=0,Type=Flag,Description="uncertain significance.https://www.ensembl.org/info/genome/variation/phenotype/phenotype_annotation.html#clin_significance">
+##INFO=<ID=known_E_1000G,Number=0,Type=Flag,Description="1000Genomes.https://www.ensembl.org/info/genome/variation/prediction/variant_quality.html#evidence_status">
+##INFO=<ID=known_E_Multiple_observations,Number=0,Type=Flag,Description="Multiple_observations.https://www.ensembl.org/info/genome/variation/prediction/variant_quality.html#evidence_status">
+##INFO=<ID=known_COSMIC_89,Number=0,Type=Flag,Description="Somatic mutations found in human cancers from the COSMIC catalogue">
+##INFO=<ID=known_CLIN_pathogenic,Number=0,Type=Flag,Description="pathogenic.https://www.ensembl.org/info/genome/variation/phenotype/phenotype_annotation.html#clin_significance">
+##INFO=<ID=known_ESP_20141103,Number=0,Type=Flag,Description="Data from NHLBI ESP version v.0.0.30. The goal of the NHLBI GO Exome Sequencing Project is to discover novel genes and mechanisms contributing to heart, lung and blood disorders by sequencing the protein coding regions of the human genome.">
+##INFO=<ID=known_CLIN_confers_sensitivity,Number=0,Type=Flag,Description="confers sensitivity.https://www.ensembl.org/info/genome/variation/phenotype/phenotype_annotation.html#clin_significance">
+##INFO=<ID=known_E_Phenotype_or_Disease,Number=0,Type=Flag,Description="Phenotype_or_Disease.https://www.ensembl.org/info/genome/variation/prediction/variant_quality.html#evidence_status">
+##INFO=<ID=known_E_ExAC,Number=0,Type=Flag,Description="ExAC.https://www.ensembl.org/info/genome/variation/prediction/variant_quality.html#evidence_status">
+##INFO=<ID=known_CLIN_not_provided,Number=0,Type=Flag,Description="not provided.https://www.ensembl.org/info/genome/variation/phenotype/phenotype_annotation.html#clin_significance">
+##INFO=<ID=dgiDB_drugs,Number=.,Type=String,Description="Combination of gene, drug, interaction types extracted from dgiDB. Each combination is pipe-seperated annotated as GENE|DRUG|TYPE">
+#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	tumor
+1	26696985	rs1283126655	C	T	30	PASS	SVLEN=.;PROB_ARTIFACT=8.16365;PROB_FFPE_ARTIFACT=inf;PROB_PRESENT=1.15636;PROB_ABSENT=10.908;ANN=T|synonymous_variant|LOW|ARID1A|ENSG00000117713|Transcript|ENST00000324856|protein_coding|1/20||ENST00000324856.13:c.582C>T|ENSP00000320485.7:p.Pro194%3D|971|582|194|P|ccC/ccT|rs1283126655||1||SNV|HGNC|HGNC:11110|YES|NM_006015.6|1|P3|CCDS285.1|ENSP00000320485|O14497||UPI0000167B91|1|||PANTHER:PTHR12656&PANTHER:PTHR12656:SF12&MobiDB_lite:mobidb-lite|||||||||||0|0|0|0|0|0|0|0|0|0|gnomAD_AFR&gnomAD_AMR&gnomAD_ASJ&gnomAD_EAS&gnomAD_FIN&gnomAD_NFE&gnomAD_OTH&gnomAD_SAS|||||||||0.0215|likely_pathogenic|0.9974	GT	./.
```

Comment:
I'd like to see testcases with multiple annotations per record, so we can check the "minimal key across annotations" choice thing.

Reply:
Regarding this specific one which uses the 'Consequence': I assume these are treated as strings (because they also inherit from string, and that provides the necessary ordering related _functions).
Could we add an example (also to the docs) that allows to sort by an order of strings specified through `--aux` (though that may be a set, not a list… not sure atm)? You can probably ignore this comment

Reply:
For now, I thought that people could compute the appropriate keys for strings within their expression. Might make the expression long of course. But that again might be mitigated by a follow-up idea for all subcommands, which would be the ability to specify a python module that can offer helper functions containing more complex reusable code. E.g. a function like

```python
def consequence_key():
    match ANN["Consequence"]:
        case "missense_variant": 0
        case "synonymous_variant: 1
```
```